docker-compose/monitor/docker-compose-v2.yml

```
@@ -0,0 +1,54 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    image: jaegertracing/jaeger:${JAEGER_IMAGE_TAG:-latest}
+    volumes:
+      - "./jaeger-ui.json:/etc/jaeger/jaeger-ui.json" # Do we need this for v2 ? Seems to be running without this.
+      - "./jaeger-v2-config.yml:/etc/jaeger/jaeger-v2-config.yml"
+    command: ["--config", "/etc/jaeger/jaeger-v2-config.yml"]
```

```suggestion
      - "./jaeger-v2-config.yml:/etc/jaeger/config.yml"
    command: ["--config", "/etc/jaeger/config.yml"]
```
Don't include unnecessary information